### PR TITLE
feat: Add accordion test wrapper

### PR DESCRIPTION
**Pull request by caalador**
Part of #1371

### CODE REVIEW

File: vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/accordion/AccordionWrapTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    @BeforeEach
+    public void registerView() {
+        RouteConfiguration.forApplicationScope()
+                .setAnnotatedRoute(AccordionView.class);
+        view = navigate(AccordionView.class);
+    }
+
+    @Test
+    void bySummary_returnsCorrectPanel() {
```
Comment (mcollovati) [inline vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/accordion/AccordionWrapTest.java:36]: Should test name be `getPanel_returnsCorrectPanel`?

File: vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/accordion/AccordionWrapTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    @Test
+    void hasPanel_returnsTrueForExistingPanel() {
+        final AccordionWrap<Accordion> wrap = wrap(view.accordion);
+        Assertions.assertTrue(wrap.hasPanel("Green"),
+                "Green panel should exist");
+        Assertions.assertFalse(wrap.hasPanel("Orange"),
+                "No Orange panel is added");
+    }
+
+}
```
Comment (mcollovati) [inline vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/accordion/AccordionWrapTest.java:74]: newline missing

File: vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/accordion/AccordionWrapTest.java (hunk 1 above)
Comment (caalador) [reply vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/accordion/AccordionWrapTest.java]: changed to getPanelBySummary_returnsCorrectPanel

File: vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/accordion/AccordionWrapTest.java (hunk 2 above)
Comment (caalador) [reply vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/accordion/AccordionWrapTest.java]: added

**Review by mcollovati — APPROVED** [review]

**Pull request merged**